optaplanner-benchmark/src/main/java/org/optaplanner/benchmark/impl/report/BenchmarkReport.java

```diff
@@ -220,11 +221,15 @@ public void writeReport() {
         for (ProblemBenchmarkResult problemBenchmarkResult : plannerBenchmarkResult.getUnifiedProblemBenchmarkResultList()) {
             if (problemBenchmarkResult.hasAnySuccess()) {
                 for (ProblemStatistic problemStatistic : problemBenchmarkResult.getProblemStatisticList()) {
+                    for (SingleStatistic singleStatistic : problemStatistic.getSingleStatisticList()) {
+                        singleStatistic.retrievePointList();
+                    }
                     problemStatistic.writeGraphFiles(this);
                 }
                 for (SingleBenchmarkResult singleBenchmarkResult : problemBenchmarkResult.getSingleBenchmarkResultList()) {
                     if (singleBenchmarkResult.isSuccess()) {
                         for (PureSingleStatistic pureSingleStatistic : singleBenchmarkResult.getPureSingleStatisticList()) {
+                            pureSingleStatistic.retrievePointList();
                             pureSingleStatistic.writeGraphFiles(this);
```

Comment:
Good question. This could actually occur, but is it realistic? Even on large benchmarks (61 singles, 10min each, 5+ single statistics each), all the pointLists have about 400MB on disk (=> not much more in memory AFAIK). On 5 threads and large datasets, the memory usage is almost 4GB at times.

Of course, in a single threaded benchmark, with small datasets that run for a long time and with a low-memory environment, this could become an issue. But the time it takes to write/read the pointLists to/from disk repeatedly might be quite a time killer in all the other cases.

Will try to run a few benchmarks to see how this turns out.